nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestConvertRecord.java

```diff
@@ -177,7 +176,7 @@ public void testWriteFailure() throws InitializationException {
         // Original FlowFile should be routed to 'failure' relationship without modification
         runner.assertAllFlowFilesTransferred(ConvertRecord.REL_FAILURE, 1);
         final MockFlowFile out = runner.getFlowFilesForRelationship(ConvertRecord.REL_FAILURE).get(0);
-        assertTrue(original == out);
+        out.assertContentEquals(original.toByteArray());
```

Comment:
Same here, can you add an assert here to verify you got the attribute set as expected?